sh/prepareData_C2HDM.py

```diff
@@ -6,29 +6,41 @@
 import pandas as pd
 import sys
 
-####### The parameters Type, Lambda1 to Lambda4, re_Lambda5, im_Lambda5, tanbeta and re_m12squared should have the label
+####### The parameters Type,Lambda1 to Lambda4,re_Lambda5,im_Lambda5, tanbeta and re_m12squared should have the label
 ####### of the corresponding parameter. With Seperator you have to tell which seperator your data file
 ####### is using (e.g. , \t or space). Your InputFILE will then be saved to OutputFILE.
 
+Seperator = "\t"
+InputFILE = "../example/C2HDM_Input.dat"
+OutputFILE = "C2HDM_Ordered.dat"
+Type = "Type"
+Lambda1 = "L1"
+Lambda2 = "L2"
+Lambda3 = "L3"
+Lambda4 = "L4"
+re_Lambda5 = "re_L5"
+im_Lambda5 = "im_L5"
+tanbeta = "tbeta"
+re_m12squared = "re_m12sq"
 
-def convert(InputFile, OutputFile):
-    print(f"Reading {InputFile}.")
-    print(f"Output is saved to {OutputFile}.")
+    HasIndexCol=False
+    Separator='\t'
+    Type='yuktype'
+    Lambda1='L1'
+    Lambda2='L2'
+    Lambda3='L3'
+    Lambda4='L4'
+    re_Lambda5='re_L5'
+    im_Lambda5='im_L5'
+    tanbeta='tbeta'
+    re_m12squared='re_m12sq'
```

Comment:
This does not work.
Executing black on this file yields errors.
All variables are defined twice, sometimes with different values.

Reply:
I propose to fix the `sh/prepareData_*.py` files by merging #165. Ok?